cmd/acra-translator/server/server.go

```diff
@@ -298,6 +312,29 @@ func (server *ReaderServer) Start(parentContext context.Context) {
 		}()
 	}
 	<-parentContext.Done()
+
+	// global 'cancel' has been called. Now we should wait (not more than specified duration) until all
+	// background goroutines spawned by readerServer will finish their execution
+	if timeoutExpired(&server.waitGroup, time.Second) {
```

Comment:
I'd extract `time.Second` into a constant like this
```go
// time to wait until goroutine shutdown
const terminationTimeout = time.Second
```
and put it somewhere at the beginning of the file so that it's easy to tweak if needed.

Reply:
+1

Reply:
Done in https://github.com/cossacklabs/acra/pull/406/commits/55e177ef6709e55c8d819195c9989dda72a96d1f